### PR TITLE
DO NOT MERGE: Add InitalDelayMs to function spec and use in sidecar

### PR DESCRIPTION
**Pull request by dturanski**
Addresses #572

### CODE REVIEW

File: function-controller/pkg/controller/deployer.go [hunk 1]
```diff
@@ -121,12 +122,18 @@ func (d *deployer) buildSidecarContainer(function *v1.Function) corev1.Container
 	if outputDestination == "" {
 		outputDestination = "replies"
 	}
+	initialDelayVal := "0"
+	if function.Spec.InitialDelayMs != nil {
```
Comment (ericbottard) [inline function-controller/pkg/controller/deployer.go]: Sorry, I was wrong in the call. If the Function went through settings defaults, `0` should already be there.
Better not to encode the logic of that default in several places...

File: function-controller/pkg/controller/deployer.go [hunk 2]
```diff
@@ -121,12 +122,18 @@ func (d *deployer) buildSidecarContainer(function *v1.Function) corev1.Container
...
+		initialDelayVal = strconv.Itoa(int(*function.Spec.InitialDelayMs))
+	}
+	//Check for nil and convert int32 to int and then to str.
```
Comment (ericbottard) [inline function-controller/pkg/controller/deployer.go]: Can remove

File: function-sidecar/cmd/function-sidecar.go [hunk 3]
```diff
@@ -94,7 +95,7 @@ func main() {
 		output = ""
 	}
 
-	log.Printf("Sidecar for function '%v' (%v->%v) using %v dispatcher starting\n", group, input, output, protocol)
+	log.Printf("Sidecar for function '%v' (%v->%v) using %v on port %v dispatcher starting\n", group, input, output, protocol, port)
```
Comment (ericbottard) [inline function-sidecar/cmd/function-sidecar.go]: The sentence now doesn't make sense (word "dispatcher" needs to be next to the protocol)

**Comment by dturanski** [comment]
Closing in favor of #577

**Pull request closed without merging**